internal/stores/contracts.go

```diff
@@ -41,6 +41,20 @@ type (
 		UploadSpending      *big.Int             `gorm:"type:bytes;serializer:gob"`
 		DownloadSpending    *big.Int             `gorm:"type:bytes;serializer:gob"`
 		FundAccountSpending *big.Int             `gorm:"type:bytes;serializer:gob"`
+
+		Sets []dbContractSet `gorm:"many2many:contract_set_contracts"`
```

Comment:
Do you ever use that field? If not I'd just remove it because I think you shouldn't need to specify it for the many2many relation to work.

Reply:
you are right, I was originally planning to drop the other side, but then realised we need the other side when we set the contracts of a contract set - so essentially a leftover